reports/agent-that-syncs-pods-and-google-sheets.md

```diff
@@ -0,0 +1,178 @@
+<!--
+Fill in the WebIDs of the people below.
+Leave this in comments!
+It's possible to have multiple people per role.
+
+Challenge/scenario creator:
+  - https://pieterheyvaert.com/#me
+Solution creator:
+  - https://pieterheyvaert.com/#me
+Report writer:
+  - https://pieterheyvaert.com/#me
+-->
+
+# Agent that syncs pods and Google Sheets
+
+The corresponding challenge is [#120](https://github.com/SolidLabResearch/Challenges/issues/120),
+which contributes to scenario [#119](https://github.com/SolidLabResearch/Challenges/issues/119).
+
+## Problem
+<!--
+You can reuse the pitch of the challenge, but check if you need to make changes.
+For example, it might happen that the approved solution does more than what the original pitch requested.
+-->
+
+A lot of users use Google Sheets to view and edit data. 
+This data can be stored in Google Sheets itself or 
+it can be coming from an external data source such as a 
+[MySQL database](https://coefficient.io/how-to-connect-mysql-database-to-google-sheets) or 
+[MongoDB](https://hightouch.com/integrations/google-sheets-source-to-mongodb). 
+So far, there is no solution that exists that allows doing this with data in Solid pods. 
+Challenge [#120](https://github.com/SolidLabResearch/Challenges/issues/120) 
+focuses on the agent/backend part of a solution for this problem.
+
+## Approved solution
+<!--
+Provide information about the approved solution:
+names of tools/libraries created, repos, and so on.
+-->
+
+We created an [agent](https://github.com/SolidLabResearch/google-sheet-sync/releases/tag/v1.0.0) that 
```

Comment:
Don't you need to include you also created an auth server to handle all the 'get google api key' shenanigans?

Reply:
and that you rely on an external rml api?

Reply:
why was all code written from scratch instead of reusing/improving existing libraries?